daliuge-engine/dlg/manager/session.py

```diff
@@ -97,6 +98,30 @@ def __init__(self, sessionId, nm=None):
         self._nm = nm
         self._dropsubs = {}
 
+        class SessionFilter(logging.Filter):
+            def __init__(self, sessionId):
+                self.sessionId = sessionId
+
+            def filter(self, record):
+                if hasattr(record, 'session_id'):
+                    return record.session_id == self.sessionId
+                return False
+
+        fmt = '%(asctime)-15s [%(levelname)5.5s] [%(threadName)15.15s] '
+        fmt += '[%(session_id)10.10s] [%(drop_uid)10.10s] '
```

Comment:
Here I'd remove appending the session_id, as it's already in the name of the file.

Reply:
Done